source/features/repo-age.tsx

```diff
@@ -15,15 +15,21 @@ const dateFormatter = new Intl.DateTimeFormat('en-US', {
 });
 
 const getRepoCreationDate = cache.function(async (): Promise<string> => {
-	const {repository} = await api.v4(`
-		repository(${getRepoGQL()}) {
-			createdAt
-		}
-	`);
+	const commitsCount = Number(select('li.commits .num')!.textContent!.trim()!.replace(',', ''));
```

Comment:
Works without trimming. Also the selectos is more specific

```suggestion
	const commitsCount = Number(select('.octicon-history + span')!.textContent!.replace(',', ''));
```

Reply:
My bad, I was wrong about the selector. Yours was fine

Reply:
Yeah, That's what I thought. Current selector is contextual and less error-prone.